File: Estrutura_Sequencial/C/Ex1020/main.c
#include <stdio.h>

int main()
{
    int idade;
    scanf("%d", &idade);

    int ano = idade / 365;
    idade %= 365;

    int mes = idade / 30;
    int dias = idade % 30;

    printf("%d ano(s)\n", ano);
    printf("%d mes(es)\n", mes);
    printf("%d dia(s)\n", dias);

    return 0;
}

File: Estrutura_Sequencial/Java/Ex1020/Main.java
package Ex1020;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int idadeDias = sc.nextInt();

        int anos = idadeDias / 365;
        idadeDias %= 365;
        int meses = idadeDias / 30;
        idadeDias %= 30;
        int dias = idadeDias;

        System.out.println(anos + " ano(s)");
        System.out.println(meses + " mes(es)");
        System.out.println(dias + " dia(s)");

        sc.close();
    }
}